manual/src/server/sal.md

```diff
@@ -0,0 +1,789 @@
+```
+
+### CancelFunkwhaleDeployment
+- Cancel the funkwhale deployment
+
+```
+func (c *Client) CancelFunkwhaleDeployment(name string) error
+```
+
+### DeployPeertube
+- Deploy vm that has peertube on it
+
+```
+func (c *Client) DeployPeertube(args tfgridBase.Peertube) (tfgridBase.PeertubeDeployment, error)
+```
+
+### GetPeertubeDeployment
+- Retrieve information about the peertube deployment (ip, etc)
+
+```
+func (c *Client) GetPeertubeDeployment(name string) (tfgridBase.PeertubeDeployment, error)
+```
+
+### CancelPeertubeDeployment
+- Cancel the discource deployment
+
+
+```
+func (c *Client) CancelPeertubeDeployment(name string) error
+```
+
+### DeployPresearch
+- Deploy vm that has presearch on it
+
+```
+func (c *Client) DeployPresearch(args tfgridBase.Presearch) (tfgridBase.PresearchDeployment, error)
+```
+
+### GetPresearchDeployment
+- Retrieve information about the presearch deployment (ip, etc)
+
+```
+func (c *Client) GetPresearchDeployment(name string) (tfgridBase.PresearchDeployment, error)
+```
+
+### CancelPresearchDeployment
+- Cancel the discource deployment
+
+```
+func (c *Client) CancelPresearchDeployment(name string) error
+```
+
+### DeployTaiga
+- Deploy vm that has taiga on it
+
+```
+func (c *Client) DeployTaiga(taiga tfgridBase.Taiga) (tfgridBase.TaigaDeployment, error) 
+```
+### GetTaigaDeployment
+- Retrieve information about the taiga deployment (ip, etc)
+
+```
+func (c *Client) GetTaigaDeployment(taigaName string) (tfgridBase.TaigaResult, error)
+```
+
+### CancelTaigaDeployment
+- Cancel the discource deployment
+
+```
+func (c *Client) CancelTaigaDeployment(name string) error 
+```
+
+## Stellar
+
+### Load
+- loads key
+- chooses network
+- starts session and creates state for client
+
+```
+func (c *Client) Load(args Load) error
+```
+
+### CreateAccount
+- Creates account on stellar given the network
+
+```
+func (c *Client) CreateAccount(network string) (string, error)
+```
+
+### Address
+- Get the stellar public address of the loaded secret
+
+```
+func (c *Client) Address() (string, error)
+```
+
+### Transactions
+- Get the last transactions of your account
+
+```
+func (c *Client) Transactions(args Transactions) ([]horizon.Transaction, error)
+```
+
+### AccountData
+- Get data related to a stellar account, leave account empty for account data of loaded account
+
+```
+func (c *Client) AccountData(account string) (horizon.Account, error)
+```
+
+### Swap
+- Swap tokens from one asset to the other (xlm to tft, etc)
+
+```
+func (c *Client) Swap(args Swap) error
+```
+
+### Transfer
+- Transfer tokens from one account to the other on stellar chain
+
+```
+func (c *Client) Transfer(args Transfer) (string, error)
+```
+
+### Balance
+- Get the balance of a specific address
+- If address is empty the balance of the loaded account will be returned
+
+```
+func (c *Client) Balance(address string) (string, error)
```

Comment:
This balance in What asset?

Reply:
Nice catch. We should allow the user to provide the asset in the call. I'll update it.

Reply:
in python client there was a list of balances being returned that has balance object (balance.balance and balance.asset), but please see the best approach possible for you :)